pythonforandroid/bootstraps/webview/__init__.py

```diff
@@ -21,31 +21,28 @@ def assemble_distribution(self):
             with open('local.properties', 'w') as fileh:
                 fileh.write('sdk.dir={}'.format(self.ctx.sdk_dir))
 
-        arch = self.ctx.archs[0]
-        if len(self.ctx.archs) > 1:
-            raise ValueError('built for more than one arch, but bootstrap cannot handle that yet')
-        info('Bootstrap running with arch {}'.format(arch))
-
         with current_directory(self.dist_dir):
             info('Copying python distribution')
 
-            self.distribute_libs(arch, [self.ctx.get_libs_dir(arch.arch)])
-            self.distribute_aars(arch)
             self.distribute_javaclasses(self.ctx.javaclass_dir,
                                         dest_dir=join("src", "main", "java"))
 
-            python_bundle_dir = join(f'_python_bundle__{arch.arch}', '_python_bundle')
-            ensure_dir(python_bundle_dir)
-            site_packages_dir = self.ctx.python_recipe.create_python_bundle(
-                join(self.dist_dir, python_bundle_dir), arch)
+            for arch in self.ctx.archs:
+                self.distribute_libs(arch, [self.ctx.get_libs_dir(arch.arch)])
+                self.distribute_aars(arch)
```

Comment:
The sdl2 bootstrap doesn't have this line but this was already here, so it's fine.

The sdl2 one also has this line which might be good to add: `ensure_dir(python_bundle_dir)`

Reply:
Yeah, hilarity in p4a. Apparently it offers the ability to distribute a pre-built AAR (library) with the app. However, someone just dropped it for seemingly unrelated reasons in dc3cc3ca51282b632f5df4797cbbd41b1594ef62.

I looked at what it does for a bit and I think it would still work after this change. Of course, we're not try to distribute AARs with our app, so I don't know if it really works or not.